### DISCUSSION TITLE
Help.....

### DISCUSSION THREAD

**Discussion by fenglilinglegeluan** [w2sv/FileNavigator#23] [Q&A]
I'm having trouble using app on my Android 11 device, system version is MIUI 12.5. I've granted the file management permission, and I've successfully run the app. I've also disabled the background restriction. However, when I try to take a photo or save an image, the app doesn't respond.

**Comment by w2sv** [comment]
Sorry about that. I assume you have the 'Image' file type and the corresponding sources (Camera, Screenshot, Download, Other Apps) enabled?

**Comment by boiling-boy-bk** [comment]
I'm having same issue but with all file types. It's not moving files. When you save the changes and go to the run screen and hit run a notification sound is produced but nothing else. It's probably not good for oxygen or later androids. Waste of time.
